antora-ui-camel/src/css/docs.css

```diff
@@ -34,6 +34,11 @@
   margin: 1.5rem;
 }
 
+.section .logo a,
+.camel-project .logo a {
+  background: none;
+}
```

Comment:
I'm sure we'll forget to wrap the link with image in a `<div class="logo">`, I think we could use `:only-child` CSS selector so we don't have to, something like `a img:only-child` instead.

Reply:
This doesn't work as I have to select the parent, though I have found the fix for it and done the required changes.